docs/using-the-jdbc-driver/UsingTheJdbcDriver.md

```diff
@@ -392,3 +393,18 @@ public static void main(String[] args) throws SQLException {
 
 ## Enable Logging
 To enable logging in the AWS JDBC Driver, change the `logger=StandardLogger` parameter to `wrapperLoggerLevel=FINEST`
+
+
+## Enable Third Party Classes and Packages
+A user application may require a special handling for particular classes or Java packages. In such case, wrapping of data objects by the AWS Advanced JDBC Driver might be crucial for a user application. The driver allows to specify classes and packages that should be left intact with no wrapping. Use the following code snippet as an example to register classes or an entire package if needed.
```

Comment:
The first two sentences might be a bit confusing due to lack of context, what do you think of
```suggestion
The AWS JDBC Driver performs special handling before or after operating on a database connection. For example, the IAM Authentication plugin generates a special IAM authentication token before establishing a connection. This is achievable by the AWS JDBC Driver wrapper over relevant data objects.

Not all data objects require special handling. Therefore, they can be left unwrapped. An example of this are classes from the PGVector package.

The driver allows you to specify classes and packages that should be left intact with no wrapping. Use the following code snippet as an example to register classes or an entire package if needed.
```